app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class ReportsController < ApplicationController
+  before_action :set_report, only: %i[show edit update destroy]
+  before_action :correct_user, only: %i[edit update destroy]
+
+  # GET /reports or /reports.json
+  def index
+    @reports = Report.order(created_at: 'DESC').all.page(params[:page])
```

Comment:
```suggestion
    @reports = Report.order(created_at: 'DESC').page(params[:page])
```

orderを付けるとallは不要になります

Reply:
取得条件が必要なのかと思っていました。
全件の場合、`order`使用なら`all`は不要なのですね。